homeassistant/components/matter/models.py

```diff
@@ -11,7 +11,8 @@
 from matter_server.client.models.node import MatterEndpoint
 
 from homeassistant.const import Platform
-from homeassistant.helpers.entity import EntityDescription
+
+from .entity import MatterEntityDescription
```

Comment:
Switch to using MatterEntityDescription instead of EntityDescription so I could access the additional attribute ignore_matter_label_names.